tests/facts/apk.ApkPackages/packages_2.json

```diff
@@ -0,0 +1,576 @@
+{
```

Comment:
There's already a [test](https://github.com/pyinfra-dev/pyinfra/blob/3.x/tests/facts/apk.ApkPackages/packages.json) for the same fact. I've chosen to keep the existing test.